tests/storage/lvm_test.py

```diff
@@ -1023,6 +1023,56 @@ def test_vg_create_multiple_devices(tmp_storage):
         assert pv.vg_name == ""
 
 
+@requires_root
+@pytest.mark.root
+def test_vg_remove_by_uuid(tmp_storage):
+    dev_size = 10 * GiB
+    num_vgs = 2
+    devs = [tmp_storage.create_device(dev_size) for _ in range(0, num_vgs)]
+    vg_names = [str(uuid.uuid4()) for _ in range(0, num_vgs)]
+
+    # TODO: should work also in read-only mode.
+    for idx, vg_name in enumerate(vg_names):
+        lvm.createVG(vg_name, [devs[idx]], f"initial-tag-{idx}", 128)
+
```

Comment:
Code above is too complicated. If we create 2 devices and 2 vgs, there to make the test more complicated than:

    dev1 = tmp_storage.create_device(dev_size)
    dev2 = tmp_storage.create_device(dev_size)

    lvm.createVG("vg-1", [dev1], f"initial-tag", 128)
    lvm.createVG("vg-2", [dev2], f"initial-tag", 128)
    
The initial tag is really the same for all vgs, no need to create unique tags.
Check how we use the intial tag in the real code.